### PR TITLE
fix: update doc, add pagination spread props

### PR DESCRIPTION
**Pull request by chrismanciero**
include displayTotalProps and linkProps in the documentation for Pagination component

### CODE REVIEW

File: src/Pagination/Pagination.Component.js [hunk 1]
```diff
@@ -45,6 +45,21 @@ export const PaginationComponent = () => {
                         description:
                             'number (default: page 1) - Initial page to be selected.'
                     },
+                    {
+                        name: 'displayTotalProps',
+                        description:
+                            'object - additional props to be spread to the display total section'
+                    },
+                    {
+                        name: 'linkProps',
```
Comment (greg-a-smith) [inline src/Pagination/Pagination.Component.js]: I missed the original PR (#224), but should we have two separate props for the "previous" and "next" links?

Comment (chrismanciero) [reply src/Pagination/Pagination.Component.js]: Separate props as in a prevNextProps which is different than the linkProps or as in prevProps and nextProps

Comment (greg-a-smith) [reply src/Pagination/Pagination.Component.js]: I was asking about `prevProps` and `nextProps`.

Comment (chrismanciero) [reply src/Pagination/Pagination.Component.js]: ok, i just added a prevProps and nextProps values to the pagination control

**Pull request merged**